cartography/data/indexes.cypher

```diff
@@ -112,6 +112,7 @@ CREATE INDEX ON :OktaTrustedOrigin(id);
 CREATE INDEX ON :OktaAdministrationRole(id);
 CREATE INDEX ON :Package(id);
 CREATE INDEX ON :Package(name);
+CREATE INDEX ON :PeeringConnection(id);
```

Comment:
Nit: should we distinguish this somehow as an AWSPeeringConnection? Does this concept exist in other clouds?